lib/search/query_components/autocomplete.rb

```diff
@@ -1,30 +1,15 @@
 module QueryComponents
   class Autocomplete < BaseComponent
-    AUTOCOMPLETE_FIELD = "title.edgengram".freeze
+    AUTOCOMPLETE_FIELD = "autocomplete".freeze
 
     def payload
       {
-        "bool" => {
-          "must" => {
-            "match" => {
-              AUTOCOMPLETE_FIELD => {
-                "query" => search_term,
-                "operator" => "and",
-              },
-            },
-          },
-          "must_not" => {
-            # The below are excluded from any autocomplete suggestions
-            # If modified remember to update autocomplete_spec!
-            "terms" => {
-              "format" => [
-                #As referenced from config/govuk_index/mapped_document_types.yaml
-                "hmrc_manual_section",
```

Comment:
Is it worth us trying to still exclude these types from the autocomplete?

Reply:
@sihugh in testing they simply didn't come up, our scoring system seemingly excludes them from all site search, and it would be beneficial to allow them in the system for future contextual searches (which is apparently supported by the ES API: https://www.elastic.co/guide/en/elasticsearch/reference/6.7/suggester-context.html )